src/components/Dialog/Dialog.js

```diff
@@ -26,15 +26,28 @@ function Dialog(props) {
   const dispatch = useDispatch();
   const { body, isModal } = props;
 
-  const escFunction = () => {
-    dispatch(setDialogVisible());
-  };
+  const dialogRef = useRef(null);
+  let activeElement;
+
+  const escFunction = e => e.which === 27 && dispatch(setDialogVisible());
 
   const registerEsc = isRegister => {
     if (isRegister) {
       document.addEventListener('keydown', escFunction, false);
+
+      activeElement = document.activeElement;
```

Comment:
Is it possible to add a test for this?

Reply:
It seems rather hard to test and this is part of the release plan for tomorrow

Reply:
@friedjoff Would it make sense to rather test this with cypress?

Reply:
Ok, let's use cypress :)